plugins/scaffolder-backend-module-http-request/.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Is this needed?

Reply:
Maybe not but it was included in a few other plugins.